### ISSUE TITLE
[Feature] Adding a possibility to continue to display each feed grouped by date

### ISSUE THREAD

**Issue by Yarg-mirror** [FreshRSS/FreshRSS#6953]
Hello,

**Is your feature request related to a problem? Please describe.**
While trying to sort "old" news of unread content I was wondering if there was a way to group them by date instead of "the 12 articles of today", "the 5 of yesterday" and "the 3,5k from the past"

**Describe the solution you’d like**
It is mostly visual I guess? But being able to keep the grouping by date might help for old/heavy content to not be so overwhelming I think

**Describe alternatives you’ve considered**
I tried looking for the option if I missed it, or if the topic have already been suggested, same if a plugin that was doing that existed, but I did not find anything, maybe I missed it?

Thanks for you time and considering that idea!

**Comment by Alkarex** [comment]
Yes, that is a reasonable request, and it should not be too difficult to add. PR welcome

**Comment by khalifat3** [comment]
Hi, Can I please work on this issue?

**Comment by Alkarex** [comment]
Implemented in:
* https://github.com/FreshRSS/FreshRSS/pull/8174

Related:
* https://github.com/FreshRSS/FreshRSS/pull/8294
